.travis.yml

```diff
@@ -60,7 +69,15 @@ script:
     - source multibuild/travis_steps.sh
     - source extra_functions.sh
     - setup_test_venv
-    - install_run $PLAT
+    # pandas-dev/pandas#41935, we get weird numpy
+    # failures on python 3.10
```

Comment:
This is not failing on pandas ci?

Reply:
It was, but we just made the CI allowed to fail. The failed tests are visible when you look at the logs of any Python 3.10 run I believe.

Reply:
> we just made the CI allowed to fail

maybe the failing tests should be XFAILed instead?

Reply:
There are a lot of flaky ones unfortunately, which is why the build is just allowed to fail.